parse/grammar/KuneiformParser.g4

```diff
@@ -127,14 +127,8 @@ typed_variable_list:
 ;
 
 constraint:
-    MIN LPAREN literal RPAREN           # min_constraint
-    | MAX LPAREN literal RPAREN         # max_constraint
-    | MINLEN LPAREN literal RPAREN      # min_len_constraint
-    | MAXLEN LPAREN literal RPAREN      # max_len_constraint
-    | (NOTNULL|NOT NULL)                # not_null_constraint
-    | (LEGACY_PRIMARY_KEY|PRIMARY KEY?) # primary_key_constraint
-    | DEFAULT LPAREN literal RPAREN     # default_constraint
-    | UNIQUE                            # unique_constraint
+    // conditionally allow some tokens, since they are used elsewhere
+    (IDENTIFIER| PRIMARY KEY? | NOT NULL | DEFAULT | UNIQUE) (LPAREN literal RPAREN)?
```

Comment:
seems no need to declare `DEFAULT` & `UNIQUE` here, in this rule we only need either a single word `IDENTIFIER` or multi word keywords.

Reply:
UNIQUE and DEFAULT are both tokens used else where. DEFAULT is used in foreign keys: `on update set default`, and unique is used in indexes: `#idx unique(column1, column2)`

Reply:
ah I just mean for `constrains` rule, since we check the 'syntax' later, but nbd